src/devices/Mcp23xxx/Mcp23xxx.cs

```diff
@@ -314,20 +304,16 @@ public void SetPinMode(int pinNumber, PinMode mode)
         public PinValue Read(int pinNumber)
         {
             ValidatePin(pinNumber);
-            Span<PinValuePair> values = stackalloc PinValuePair[1];
-            values[0] = new PinValuePair(pinNumber, default);
+            Span<PinValuePair> values = stackalloc PinValuePair[] { new PinValuePair(pinNumber, default) };
             Read(values);
             return values[0].PinValue;
         }
 
         public void Read(Span<PinValuePair> pinValues)
```

Comment:
Summary since public?

Reply:
As a general rule I try not to duplicate comments for overrides or interface implementations. Quite often they end up deviating over time and getting messy. I'm not super strict on that point and I don't know that we have actual guidelines around it.